.github/ISSUE_TEMPLATE/maintenance-task.md

```diff
@@ -7,4 +7,19 @@ assignees: ''
 
 ---
 
+### Level of urgency
+- [ ] High
```

Comment:
Maybe we could add a space for if there's a deadline for this item?  For example, an EOL for upgrading a dependency?

Reply:
Maybe that's a distinct category - [ ] EOL Date XX/XX/2023